PyViCare/PyViCareGateway.py

```diff
@@ -4,6 +4,10 @@
 
 class Gateway(Device):
 
+    @handleNotSupported
+    def getSerial(self):
+        return self.service.getProperty("gateway.devices")["gatewayId"]
```

Comment:
At least my Heatbox1 / VitoconnectOpto1 has no device.serial data point. I requested it in the [Viessmann community](https://community.viessmann.de/t5/Feedback-API/Serial-Number-an-Gateways/td-p/410253) but no reply so far. So I get the serial from another data point.